lib/utils/fs-utils.js

```diff
@@ -59,8 +59,53 @@ function existsAndNonZero(path) {
         });
 }
 
+function isSameFlavorSet(flavorsPath, flavorsSet, excludeDirs, pattern) {
+    return qio.listTree(flavorsPath,
```

Comment:
same comment as before... you have the playlist generator.... why  you need to look at the disk?